layrry-launcher/src/main/java/org/moditect/layrry/internal/LayersBuilderImpl.java

```diff
@@ -15,8 +15,12 @@
  */
 package org.moditect.layrry.internal;
 
+import java.nio.file.Path;
+import java.util.HashSet;
```

Comment:
Personally I prefer `LinkedHashSet` to keep insertion order. This class already makes use of `LinkedHashMap` 😏

Reply:
I don't think it's needed, as there's no direct relationship between different plug-in directories. A relationship _does_ exist for the derived layers, and that's why LHM is used there.